.gitignore

```diff
@@ -2,3 +2,4 @@ build/
 __pycache__
 .idea
 .tox
+.coverage
```

Comment:
Let's be careful with `.*ignore` files, they might make the diff confusing. You'd want to add these files if you are changing something in the CI or in the structure of the repo. You can keep it here for now, let's avoid this in future PRs.

Reply:
noted. I think this file is created from running unit test. I wanted to avoid committing it. It is not currently committed.